### DISCUSSION TITLE
How do I invert a vim_mode = condition for keybinding context?

### DISCUSSION THREAD

**Discussion by BenCandibots** [zed-industries/zed#19029] [Feature Requests]
I would like to create a context that encapsulate everything save for insert mode? I attempted 
- `vim_mode != insert`
- `!vim_mode == insert`
-  `! (vim_mode == insert)`

and additional variations.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
